modules/@ergonode/attributes/src/config/extends.js

```diff
@@ -2,6 +2,99 @@
  * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
  * See LICENSE for license details.
  */
+import {
+    prepareOptionsData,
+    prepareParametersData,
+    prepareTextAreaData,
+    sendOptionsData,
+    setParametersData,
+    setTextAreaData,
+} from '@Attributes/extends/methods';
+
+import {
+    Components,
+    Icons,
+    Store,
+} from './imports';
+
+const getTypeConfiguration = ({
+    $this = null, type,
+}) => {
+    switch (type) {
+    case 'DATE':
+        return {
+            params: {
+                key: 'format',
+                value: $this.state.dictionaries.dateFormats,
+                fieldName: 'parameters',
+            },
+            icon: Icons.IconDate,
+        };
+    case 'UNIT':
+        return {
+            params: {
+                key: 'unit',
+                value: $this.state.dictionaries.units,
+                fieldName: 'parameters',
+            },
+            icon: Icons.IconUnit,
+        };
+    case 'PRICE':
+        return {
+            params: {
+                key: 'currency',
+                value: $this.state.dictionaries.currencies,
+                fieldName: 'parameters',
+            },
+            icon: Icons.IconPrice,
+        };
+    case 'TEXT_AREA':
+        return {
+            params: {
+                key: 'richEdit',
+                fieldName: 'parameters',
+            },
+            icon: Icons.IconTextArea,
+        };
+    case 'TEXT':
+        return {
+            icon: Icons.IconText,
+        };
+    case 'NUMERIC':
+        return {
+            icon: Icons.Numeric,
+        };
+    case 'SELECT':
+        return {
+            params: {
+                fieldName: 'options',
+            },
+            icon: Icons.IconSelect,
+        };
+    case 'MULTI_SELECT':
+        return {
+            params: {
+                fieldName: 'options',
+            },
+            icon: Icons.IconMultiSelect,
+        };
+    case 'IMAGE':
+        return {
+            icon: Icons.IconImage,
+        };
+    case 'GALLERY':
+        return {
+            icon: null,
```

Comment:
Default icon for attribute should be `IconAttributes`

Reply:
yes, but this is only configuration for modules. This is our responsibility to set the default icon if  not set.